security/src/main/java/com/fincity/security/dao/ClientUrlDAO.java

```diff
@@ -92,4 +92,13 @@ public Mono<Boolean> checkSubDomainAvailability(String subDomain) {
 						.limit(1))
 				.map(e -> e.value1() == 0);
 	}
+
+	public Mono<ClientUrl> getClientUrl(ULong clientId, String appcode) {
```

Comment:
Vishwas just finished getclienturl in clientUrlService. Please merge master and use that.